test/RibbonThetaVaultWithSwap.ts

```diff
@@ -1540,11 +1439,108 @@ function behavesLikeRibbonOptionsVault(params: {
         await depositIntoVault(collateralAsset, vault, depositAmount);
         const res = await vault
           .connect(ownerSigner)
-          .commitAndClose({ from: owner });
+          .closeRound({ from: owner });
 
         const receipt = await res.wait();
-        assert.isAtMost(receipt.gasUsed.toNumber(), 2597051);
-        // console.log("commitAndClose", receipt.gasUsed.toNumber());
+        assert.isAtMost(receipt.gasUsed.toNumber(), 137969);
+        // console.log("closeRound", receipt.gasUsed.toNumber());
+      });
+    });
+
+    describe("#commitNextOption", () => {
+      time.revertToSnapshotAfterEach();
+
+      it("reverts when not called with keeper", async function () {
```

Comment:
We can add a test case `it("reverts when the currentOption is not closed")` and `it("sets the option correctly on the first round")`

Reply:
Okay, will do